+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Fri Feb  3 10:41:40 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL strongswan
+ pkgs=strongswan
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs strongswan --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpho2z_uqg
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpho2z_uqg
+ service=strongswan.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-02-03 10:42:16'
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Fri Feb  3 10:42:16 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Fri Feb  3 10:42:17 2023 - INFO  - Start testing...
+ systemctl restart strongswan.service
+ CHECK_RESULT 0 0 0 'strongswan.service restart failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='strongswan.service restart failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl stop strongswan.service
+ CHECK_RESULT 0 0 0 'strongswan.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='strongswan.service stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl start strongswan.service
+ CHECK_RESULT 0 0 0 'strongswan.service start failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='strongswan.service start failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status strongswan.service
+ grep Active
+ grep 'active (running)'
     Active: active (running) since Fri 2023-02-03 10:42:18 CST; 264ms ago
+ CHECK_RESULT 0 0 0 'strongswan.service start failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='strongswan.service start failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
++ systemctl enable strongswan.service
++ awk '{print $3}'
+ symlink_file=/etc/systemd/system/multi-user.target.wants/strongswan.service
+ find /etc/systemd/system/multi-user.target.wants/strongswan.service
/etc/systemd/system/multi-user.target.wants/strongswan.service
+ CHECK_RESULT 0 0 0 'strongswan.service enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='strongswan.service enable failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl disable strongswan.service
Removed /etc/systemd/system/multi-user.target.wants/strongswan.service.
+ find /etc/systemd/system/multi-user.target.wants/strongswan.service
find: ‘/etc/systemd/system/multi-user.target.wants/strongswan.service’: No such file or directory
+ CHECK_RESULT 1 0 1 'strongswan.service disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='strongswan.service disable failed'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ journalctl --since '2023-02-03 10:42:16' -u strongswan.service
+ grep -v /etc/strongswan/ipsec.d/triplets.dat
+ grep -v /etc/tnc_config
Feb 03 10:42:19 openEuler-riscv64 charon[2278]: 00[KNL] received netlink error: Operation not supported (95)
Feb 03 10:42:19 openEuler-riscv64 charon[2278]: 00[KNL] received netlink error: Operation not supported (95)
+ CHECK_RESULT 0 0 1 'There is an error message for the log of strongswan.service'
+ actual_result=0
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of strongswan.service'
+ '[' -z 0 ']'
+ '[' 1 -eq 0 ']'
+ test 0x == 0x
+ test -n 'There is an error message for the log of strongswan.service'
+ LOG_ERROR 'There is an error message for the log of strongswan.service'
+ message='There is an error message for the log of strongswan.service'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of strongswan.service'
Fri Feb  3 10:42:27 2023 - ERROR - There is an error message for the log of strongswan.service
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_strongswan.sh line 47'
+ message='oe_test_service_strongswan.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_strongswan.sh line 47'
Fri Feb  3 10:42:28 2023 - ERROR - oe_test_service_strongswan.sh line 47
+ return 0
+ test_reload strongswan.service
+ service=strongswan.service
+ systemctl start strongswan.service
+ systemctl reload strongswan.service
+ grep 'Job type reload is not applicable'
Failed to reload strongswan.service: Job type reload is not applicable for unit strongswan.service.
+ CHECK_RESULT 0 0 0 'Job type reload is not applicable for unit strongswan.service'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit strongswan.service'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status strongswan.service
+ grep 'Active: active'
     Active: active (running) since Fri 2023-02-03 10:42:18 CST; 10s ago
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Fri Feb  3 10:42:29 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 1
+ test 1 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Feb  3 10:42:30 2023 - ERROR - failed to execute the case.
+ exit 1
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Feb  3 10:42:30 2023 - INFO  - start environment cleanup.
+ systemctl stop strongswan.service
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpho2z_uqg ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpho2z_uqg
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri Feb  3 10:42:47 2023 - INFO  - Finish environment cleanup!
